src/content/release-notes.mdx

```diff
@@ -5,6 +5,43 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.19.0
+
+16 April 2024
+
+This versions is compatible with Kubernetes versions 1.25 to 1.29
+
+### New Features
+
+- Okteto Insights: Pod v2, User, Namespace, and Preview metrics are now available <!-- 7897, 7906, 7898, 7900 -->
+
+### Breaking Changes
+
+- Okteto CLI: `okteto deploy` will now ignore host volumes. To manage the mounting of volumes, use a Dockerfile with the `COPY` instruction instead of references in your Docker compose.
+
+### Improvements
+
+- Okteto CLI: [Smart Builds implementation](core/build-service.mdx) now uses `build: context` instead of `project: commit`
+- Update buildkit to v0.12.5-okteto1  <!-- 7830 -->
+- Add Grafana endpoint to `cluster/general` to support Oketeo Insights for SaaS plan users  <!-- 7861 -->
+- maxNamespaces quota should no longer affect the okteto-bot user  <!-- 7905 -->
+- Rename OKTETO_CURRENT_DEPLOY_BELONGS_TO_PREVIEW to OKTETO_IS_PREVIEW_ENVIRONMENT and allow it to be used for destroy operations  <!-- 7904 -->
```

Comment:
In this one, I would say that we added the environment variable `OKTETO_IS_PREVIEW_ENVIRONMENT` for dev environments deployed from UI and previews. `OKTETO_CURRENT_DEPLOY_BELONGS_TO_PREVIEW` was something internal and we never document nor tell customers to use it

Reply:
Ahh ok thanks for the clarification